apps/api-extractor/src/analyzer/TypeScriptHelpers.ts

```diff
@@ -123,13 +123,19 @@ export class TypeScriptHelpers {
 
   // Return name of the module, which could be like "./SomeLocalFile' or like 'external-package/entry/point'
   public static getModuleSpecifier(
-    declarationWithModuleSpecifier: ts.ImportDeclaration | ts.ExportDeclaration
+    nodeWithModuleSpecifier: ts.ImportDeclaration | ts.ExportDeclaration | ts.ImportTypeNode
   ): string | undefined {
+    if (nodeWithModuleSpecifier.kind === ts.SyntaxKind.ImportType) {
+      return ((nodeWithModuleSpecifier.argument as ts.LiteralTypeNode)
+        .literal as ts.StringLiteral).text.trim();
```

Comment:
Seems like these type casts might be making some assumptions that are not always true.

Reply:
Added an explicit assertion